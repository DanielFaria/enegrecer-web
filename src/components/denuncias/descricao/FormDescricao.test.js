import React from 'react';
import toJson from 'enzyme-to-json';
import { mount } from 'enzyme';
import Descricao from './FormDescricao';

describe('Form Descrição', () => {
  it('deve seguir a estrutura definida ', () => {
    const desricao = mount(<Descricao />);
    const desricaoJson = toJson(desricao);
    expect(desricaoJson).toMatchSnapshot();
  });
});
