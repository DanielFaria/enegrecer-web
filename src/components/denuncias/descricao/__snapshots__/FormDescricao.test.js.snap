// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form Descrição deve seguir a estrutura definida  1`] = `
<descricao>
  <div>
    <br />
    <p>
      Descrição da agressão
    </p>
    <br />
    <h4>
       Compreendemos o quanto é difícil externalizar a agressão em palavras.
    </h4>
    <p>
      Mas, é importante para que futuramente consigamos trabalhar para que essas agressões tenham um fim. 
    </p>
    <div
      className="row"
    >
      <combobox
        divClasse="input-field col s6"
        id="periodo"
        itens={
          Array [
            " ",
            "Manhã",
            "Tarde",
            "Noite",
          ]
        }
        label="Período*"
        onChange={[Function]}
        valorPadrao=""
      >
        <div
          className="input-field col s6"
        >
          <select
            id="periodo"
            onChange={[Function]}
          >
            <option
              value=""
            />
            <option
              key=" "
              value=" "
            >
               
               
               
            </option>
            <option
              key="Manhã"
              value="Manhã"
            >
               
              Manhã
               
            </option>
            <option
              key="Tarde"
              value="Tarde"
            >
               
              Tarde
               
            </option>
            <option
              key="Noite"
              value="Noite"
            >
               
              Noite
               
            </option>
          </select>
          <label
            htmlFor="periodo"
          >
            Período*
          </label>
        </div>
      </combobox>
      <Data
        divClasse="input-field col s6"
        id="data-agressao"
        label="Data de Agressão"
        onChange={[Function]}
      >
        <campoTexto
          divClasse="input-field col s6"
          id="data-agressao"
          inputClasse="datepicker"
          label="Data de Agressão"
          maxLen={0}
          onChange={[Function]}
          placeholder="Data"
          type="date"
        >
          <div
            className="input-field col s6"
          >
            <input
              className="datepicker"
              id="data-agressao"
              maxLength={0}
              onChange={[Function]}
              placeholder="Data"
              type="date"
            />
            <label
              className="active"
              htmlFor="data-agressao"
            >
              Data de Agressão
            </label>
          </div>
        </campoTexto>
      </Data>
    </div>
    <div
      className="row"
    >
      <comboEstado
        classes=" col s6"
        id="descricao-estado"
        onChange={[Function]}
      >
        <combobox
          divClasse="input-field  col s6"
          id="descricao-estado"
          itens={
            Array [
              "AC",
              "AL",
              "AM",
              "AP",
              "BA",
              "CE",
              "DF",
              "ES",
              "GO",
              "MA",
              "MG",
              "MS",
              "MT",
              "PA",
              "PB",
              "PE",
              "PI",
              "RN",
              "PR",
              "RJ",
              "RR",
              "SC",
              "SE",
              "SP",
              "TO",
            ]
          }
          label="Selecione o Estado:"
          onChange={[Function]}
          valorPadrao="Selecione"
        >
          <div
            className="input-field  col s6"
          >
            <select
              id="descricao-estado"
              onChange={[Function]}
            >
              <option
                value=""
              >
                Selecione
              </option>
              <option
                key="AC"
                value="AC"
              >
                 
                AC
                 
              </option>
              <option
                key="AL"
                value="AL"
              >
                 
                AL
                 
              </option>
              <option
                key="AM"
                value="AM"
              >
                 
                AM
                 
              </option>
              <option
                key="AP"
                value="AP"
              >
                 
                AP
                 
              </option>
              <option
                key="BA"
                value="BA"
              >
                 
                BA
                 
              </option>
              <option
                key="CE"
                value="CE"
              >
                 
                CE
                 
              </option>
              <option
                key="DF"
                value="DF"
              >
                 
                DF
                 
              </option>
              <option
                key="ES"
                value="ES"
              >
                 
                ES
                 
              </option>
              <option
                key="GO"
                value="GO"
              >
                 
                GO
                 
              </option>
              <option
                key="MA"
                value="MA"
              >
                 
                MA
                 
              </option>
              <option
                key="MG"
                value="MG"
              >
                 
                MG
                 
              </option>
              <option
                key="MS"
                value="MS"
              >
                 
                MS
                 
              </option>
              <option
                key="MT"
                value="MT"
              >
                 
                MT
                 
              </option>
              <option
                key="PA"
                value="PA"
              >
                 
                PA
                 
              </option>
              <option
                key="PB"
                value="PB"
              >
                 
                PB
                 
              </option>
              <option
                key="PE"
                value="PE"
              >
                 
                PE
                 
              </option>
              <option
                key="PI"
                value="PI"
              >
                 
                PI
                 
              </option>
              <option
                key="RN"
                value="RN"
              >
                 
                RN
                 
              </option>
              <option
                key="PR"
                value="PR"
              >
                 
                PR
                 
              </option>
              <option
                key="RJ"
                value="RJ"
              >
                 
                RJ
                 
              </option>
              <option
                key="RR"
                value="RR"
              >
                 
                RR
                 
              </option>
              <option
                key="SC"
                value="SC"
              >
                 
                SC
                 
              </option>
              <option
                key="SE"
                value="SE"
              >
                 
                SE
                 
              </option>
              <option
                key="SP"
                value="SP"
              >
                 
                SP
                 
              </option>
              <option
                key="TO"
                value="TO"
              >
                 
                TO
                 
              </option>
            </select>
            <label
              htmlFor="descricao-estado"
            >
              Selecione o Estado:
            </label>
          </div>
        </combobox>
      </comboEstado>
      <campoTexto
        divClasse="input-field col s6"
        id="cidade-descricao"
        inputClasse=""
        label="Cidade*"
        maxLen={40}
        onChange={[Function]}
        placeholder=""
        type="text"
      >
        <div
          className="input-field col s6"
        >
          <input
            className=""
            id="cidade-descricao"
            maxLength={40}
            onChange={[Function]}
            type="text"
          />
          <label
            className="active"
            htmlFor="cidade-descricao"
          >
            Cidade*
          </label>
        </div>
      </campoTexto>
    </div>
    <div
      className="row"
    >
      <campoTexto
        divClasse="input-field col s6"
        id="bairro-descricao"
        inputClasse=""
        label="Bairro*"
        maxLen={40}
        onChange={[Function]}
        placeholder=""
        type="text"
      >
        <div
          className="input-field col s6"
        >
          <input
            className=""
            id="bairro-descricao"
            maxLength={40}
            onChange={[Function]}
            type="text"
          />
          <label
            className="active"
            htmlFor="bairro-descricao"
          >
            Bairro*
          </label>
        </div>
      </campoTexto>
      <campoTexto
        divClasse="input-field col s6"
        id="endereco-descricao"
        inputClasse=""
        label="Endereço"
        maxLen={40}
        onChange={[Function]}
        placeholder=""
        type="text"
      >
        <div
          className="input-field col s6"
        >
          <input
            className=""
            id="endereco-descricao"
            maxLength={40}
            onChange={[Function]}
            type="text"
          />
          <label
            className="active"
            htmlFor="endereco-descricao"
          >
            Endereço
          </label>
        </div>
      </campoTexto>
    </div>
    <div
      className="row"
    >
      <campoTexto
        divClasse="input-field col s12"
        id="descricao-denuncia"
        inputClasse="materialize-textarea"
        label="Descrição da denúncia"
        maxLen={255}
        onChange={[Function]}
        placeholder=""
        type="text"
      >
        <div
          className="input-field col s12"
        >
          <input
            className="materialize-textarea"
            id="descricao-denuncia"
            maxLength={255}
            onChange={[Function]}
            type="text"
          />
          <label
            className="active"
            htmlFor="descricao-denuncia"
          >
            Descrição da denúncia
          </label>
        </div>
      </campoTexto>
    </div>
  </div>
</descricao>
`;
